### ISSUE TITLE
MiKo_2070 CodeFixProvider causes ArgumentOutOfRangeException 

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#370]
Following snippet produces a ArgumentOutOfRangeException:
```
        /// <summary>
        /// Returns <see langword="true"/> if restarting the application was triggered.
        /// </summary>
```

```
System.ArgumentOutOfRangeException : StartIndex cannot be less than zero.
Parameter name: startIndex
   at System.String.Substring(Int32 startIndex,Int32 length)
   at MiKoSolutions.Analyzers.CommentExtensions.WithoutFirstWord(String text)
   at MiKoSolutions.Analyzers.Rules.Documentation.MiKo_2070_CodeFixProvider.GetUpdatedSyntax(Document document,SyntaxNode syntax,Diagnostic diagnostic)
   at MiKoSolutions.Analyzers.Rules.MiKoCodeFixProvider.ApplyDocumentCodeFixAsync(Document document,SyntaxNode root,SyntaxNode syntax,Diagnostic diagnostic)
   at MiKoSolutions.Analyzers.Rules.MiKoCodeFixProvider.<>c__DisplayClass21_0.<CreateCodeFix>b__1(CancellationToken _)
   at Microsoft.CodeAnalysis.CodeActions.CodeAction.DocumentChangeAction.GetChangedDocumentAsync(CancellationToken cancellationToken)
   at async Microsoft.CodeAnalysis.CodeActions.CodeAction.GetChangedSolutionAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.CodeActions.CodeAction.ComputeOperationsAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.CodeActions.CodeAction.GetPreviewOperationsAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedAction.GetPreviewResultAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedActionWithNestedFlavors.PreviewChangesSuggestedAction.CreateAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedActionWithNestedFlavors.GetPreviewChangesFlavorAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Editor.Implementation.Suggestions.SuggestedActionWithNestedFlavors.CreateAllFlavorsAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.Extensions.IExtensionManagerExtensions.PerformFunctionAsync[T](<Unknown Parameters>)
```
